engine/physics/src/box2d/Box2D/Dynamics/b2World.h

```diff
@@ -185,6 +185,12 @@ class b2World
 	/// Get the global gravity vector.
 	b2Vec2 GetGravity() const;
 
+	/// Get physics Box2D velocity iteration.
+	int GetVelocityIteration() const;
+
+	/// Get physics Box2D position iteration.
+	int GetPositionIteration() const;
```

Comment:
These functions are redundant, as the values are _passed in_ to the b2World, from physics_2d.cpp, so you don't need to "get them" from b2world.